bundles/org.openhab.binding.elgatokeylight/src/main/java/org/openhab/binding/elgatokeylight/internal/ElgatoKeyLightConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.elgatokeylight.internal;
+
+/**
+ * The {@link ElgatoKeyLightConfiguration} class contains fields mapping thing
+ * configuration parameters.
+ *
+ * @author Gunnar Wagenknecht - Initial contribution
+ */
+public class ElgatoKeyLightConfiguration {
```

Comment:
Please add the null annotation. You could assign default values to the fields to prevent additional null checks.